lesson11/main.js

```diff
@@ -0,0 +1,88 @@
+const ul = document.getElementById("js-ul");
+
+const renderLoading = () => {
+  const loading = document.getElementById("js-loading");
+  const loadingImg = document.createElement("img");
+  loadingImg.src = "loading.gif";
+  loadingImg.id = "loading-img";
+  loading.appendChild(loadingImg);
+};
+
+const removeLoading = () => {
+  document.getElementById("loading-img").remove();
+};
+
+const renderLists = (features) => {
+  const fragment = new DocumentFragment();
+
+  for (let feature of features) {
+    const li = document.createElement("li");
+    const a = document.createElement("a");
+    const img = document.createElement("img");
+    a.href = feature.to;
+    a.textContent = feature.text;
+    img.src = feature.img;
+    img.alt = feature.alt;
+
+    fragment
+      .appendChild(li)
+      .appendChild(a)
+      .insertAdjacentElement("afterbegin", img);
+  }
+
+  ul.appendChild(fragment);
+};
+
+//Define the function to show information.
+//In this lesson, this function is used to display the passed message for unexpected errors(not include fetch error)
+const displayInfo = (info) => {
+  const li = document.createElement('li');
+  if (typeof info === 'string') {
+    li.texteContent = info;
```

Comment:
I think `textContent` is correct.
```suggestion
    li.textContent = info;
```

Reply:
Sae-san,
Thank you for your comment! It helped to solve the issue Mai-san pointed out below too.
Kindly see the latest commit: 5a85c312f80658e38492e308814b7e0b561fe0f3